query/src/main/scala/filodb/query/exec/rangefn/InstantFunction.scala

```diff
@@ -61,7 +53,7 @@ trait HDToDoubleIFunction extends HistogramInstantFunction {
     * @param value Sample against which the function will be applied
     * @return Calculated value
     */
-  def apply(h: Histogram, d: Double): Double
+  def apply(h: Histogram, d: Double, scalarParam: Seq[Double] = Nil): Double
```

Comment:
nitpick: Use plural `scalarParams` here as well as in implementations. I was wondering how there could be one param, but many values...